restconnector/RestConnector.h

```diff
@@ -11,20 +11,35 @@
 namespace restconnector
 {
 
+/**
+ * The RestConnector class is responsible starting a HTTP server
+ * in a dedicated thread, according to a given hostname and port.
+ * HTTP requests are forwarded to an implicitely registered handler.
+ */
 class RestConnector
 {
 public:
 
-    RestConnector( const std::string& address, const std::string& port );
+    /**
+     * Default constructor.
+     * @param hostname Hostname or IP address
+     * @param port Listening port
+     */
+    RestConnector( const std::string& hostname, const std::string& port );
```

Comment:
Cppnetlib expects a string, and we receive it as a string (from the command line parameters). Changing it to uint16_t would mean a double useless conversion.